client/wfnews-war/src/main/angular/src/app/services/AGOL-service.ts

```diff
@@ -174,7 +174,7 @@ export class AGOLService {
   }
 
   getAreaRestrictionsWfs () {
-    const url = 'https://openmaps.gov.bc.ca/geo/pub/WHSE_LAND_AND_NATURAL_RESOURCE.PROT_RESTRICTED_AREAS_SP/ows?service=wfs&version=1.1.0&request=GetFeature&typename=WHSE_LAND_AND_NATURAL_RESOURCE.PROT_RESTRICTED_AREAS_SP&outputFormat=application/json&SRSName=urn:x-ogc:def:crs:EPSG:4326'
+    const url = 'c?service=wfs&version=1.1.0&request=GetFeature&typename=WHSE_LAND_AND_NATURAL_RESOURCE.PROT_RESTRICTED_AREAS_SP&outputFormat=application/json&SRSName=urn:x-ogc:def:crs:EPSG:4326'
```

Comment:
was the "c" intentional?

Reply:
Good catch! This was a typo... we dont want to change this at all